charts/alfresco-connector-ms365/templates/deployment-connector-ms365.yaml

```diff
@@ -34,7 +34,8 @@ spec:
           env:
             {{- include "alfresco-connector-ms365.env" $ | indent 12 }}
           ports:
-              - containerPort: {{ .Values.image.internalPort }}
+            - name: {{ .Values.readinessProbe.tcpSocket.port }}
```

Comment:
would be a bit confusing to use this here, I would hardcode the nameport and have such value in Values.readinessProbe.tcpSocket.port